### PR TITLE
Update ja_jp.lang

### PR DESCRIPTION
**Pull request by otoma417**
濃縮チャンバーの濃縮を品質向上に修正、英語での説明やレシピなどを見るとこちらが適切と思いました。
濃縮鉄、濃縮合金の濃縮を富化に修正しました
HDPEを高密度ポリエチレンに修正
chamberはチャンバーに統一

### CODE REVIEW

File: src/main/resources/assets/mekanism/lang/ja_jp.lang [hunk 1]
```diff
@@ -1136,6 +1142,10 @@ mekanism.configgui.ctgy.tools.general.tooltip=MekanismToolsの基本設定
 mekanism.configgui.ctgy.tools.armor.tooltip=MekanismToolsの防具におけるパラメータの設定
 mekanism.configgui.ctgy.tools.tools.tooltip=MekanismToolsの道具におけるパラメータの設定
 
+# Commands
```
Comment (pupnewfster) [inline src/main/resources/assets/mekanism/lang/ja_jp.lang]: The different command lang files added here should have a value on the right side of the equals, or just not be included.

Comment (otoma417) [reply src/main/resources/assets/mekanism/lang/ja_jp.lang]: Fixed.

Comment (pupnewfster) [reply src/main/resources/assets/mekanism/lang/ja_jp.lang]: It appears to still have the "blank" entries I was talking about:
```
cmd.mek.usage=
cmd.mek.debug.usage=
cmd.mek.debug=
cmd.mek.chunk.usage=
```

Comment (otoma417) [reply src/main/resources/assets/mekanism/lang/ja_jp.lang]: I don't know what value to write there. The entry field seems to be blank for other translations. Can't I leave this blank?

Comment (pupnewfster) [reply src/main/resources/assets/mekanism/lang/ja_jp.lang]: Ah, I had not noticed/realized they are blank even in the main en_us.lang file. This is fine as is then and I will figure out what to do with them separately. Is this good to merge or are you still making other misc changes.

**Comment by otoma417** [comment]
Yes, just merge.

2019年9月19日(木) 10:30 Richard Freimer <notifications@github.com>:

**Pull request merged**